### PR TITLE
Added helpers

### PR DESCRIPTION
**Pull request by Roxedus**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-swag/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Replaced index.html with page with some info, including a link to the swag documentation.
Added http502 helper page, also linking to our docs.

UI suggestions encouraged

## Benefits of this PR and context:

Increase the possibility of self-help, decrease support requests

## How Has This Been Tested?

It builds and shows the expected pages

### CODE REVIEW

**Comment by Roxedus** [comment]
Forgot to update the date on the default file, will wait for any possible review changes to justify

File: root/defaults/502.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            color: rgb(207, 48, 139);
+        }
+        </style>
+    </head>
+    <body>
+        <div class="message">
+            <h1>502</h1>
+            <h2>Nginx can not connect to the application</h2>
+            <p>Some common reasons are listed here: <a target="_blank" href="https://docs.linuxserver.io/general/swag#502">docs.linuxserver.io</a></p>
+            <p>For help and support, please visit: <a target="_blank" href="https://www.linuxserver.io/support">inuxserver.io/support</a></p>
```
Comment (aptalca) [inline root/defaults/502.html:41]: Missing an `l` in `linuxserver`

File: root/defaults/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        a{
+            color: rgb(207, 48, 139);
+        }
+        </style>
+    </head>
+    <body>
+        <div class="message">
+            <h1>Welcome to your <a target="_blank" href="https://github.com/linuxserver/docker-swag">SWAG</a> instance</h1>
+            <p>A webserver and reverse proxy solution brought to you by <a target="_blank" href="https://www.linuxserver.io/">linuxserver.io</a> with php support and a built-in Certbot client.</p>
+            <p>We have a article on how to use swag here: <a target="_blank" href="https://docs.linuxserver.io/general/swag">docs.linuxserver.io</a></p>
```
Comment (aptalca) [inline root/defaults/index.html:35]: should be `an article`

File: root/defaults/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            <p>For help and support, please visit: <a target="_blank" href="https://www.linuxserver.io/support">inuxserver.io/support</a></p>
```
Comment (aptalca) [inline root/defaults/index.html:36]: missing an `l` in `linuxserver`

**Review by aptalca — APPROVED** [review]

**Pull request merged**